public/app/features/correlations/CorrelationsPage.tsx

```diff
@@ -157,9 +158,12 @@ export default function CorrelationsPage() {
       navModel={navModel}
       subTitle={
         <>
-          Define how data living in different data sources relates to each other. Read more in the{' '}
+          <Trans i18nKey="correlations.sub-title">
+            Define how data living in different data sources relates to each other. Read more in the
```

Comment:
Links to docs should be part of the translation. You can see an example in how `time-picker.content.empty-recent-list-docs` was introduced.